README.md

```diff
@@ -99,6 +99,9 @@ where YYYY-MM-DD is today's date.
 - General refactoring of JavaScript code
 - Changed URL Query string parsing to be order independent
 
+## Alternatives
```

Comment:
Please add spacing after headings for better readability

Reply:
Sure, did it :)